source/docs/installation.md

```diff
@@ -67,6 +67,10 @@ We can install Embark using the Node Package Manager (no worries, that one comes
 
 <pre><code class="shell">$ npm -g install embark</code></pre>
 
+If you want to have access to Embark 4 alpha, do this instead:
```

Comment:
This should be written more generic. There's gonna be a day where we'll be on Embark 5.x and 6.x might be `next`. So let's rather add a section that explains how to get whatever `next` is.

As a matter of fact, I think we should have that below the "Installing Embark" section. Something like "Installing latest development version."